addons/main/script_version.hpp

```diff
@@ -1,4 +1 @@
-#define MAJOR 3
-#define MINOR 7
-#define PATCHLVL 0
-#define BUILD 180430
+#define MAJOR 3\n#define MINOR 7\n#define PATCHLVL 0\n#define BUILD 180430
```

Comment:
This looks like a mistake

Reply:
Regex? 😆

Reply:
Who is `Roderick Mabry` and how did he get push access?

Reply:
This whole thing is quite interesting. 
I was testing out some new merge tools. 
Looks like some fixes are in order.

Reply:
This is not fixed yet ;) you only fixed the color issue. And now github shows this comment as outdated 🤔

Reply:
Micro Commits. Its coming :)